### DISCUSSION TITLE
Is it ok to close the Tgrabs window early?

### DISCUSSION THREAD

**Discussion by walsmanjason** [mooch443/trex#155] [Questions] [answered]
When I run Tgrabs on a video to create a .pv file, Tgrabs takes some time then opens a Tgrabs window which shows the video and identified blobs. This plays through the whole video, usually at faster than real time speed, then automatically closes.

I've found that I can close the Tgrabs window before it has played the whole video and the .pv file is still created. Stupid question but is there any reason to let the Tgrabs window play through the whole video (as it does by default) before closing it and moving to my next task? I would love to save time but would hate to somehow create a lower quality .pv file.

Thanks

**Comment by mooch443** [comment, marked as answer]
Hey, so the way it works it that whenever you click "close" or press CTRL+C in the terminal, the video conversion is stopped. This does not speed up the conversion, it just stops it at the point where you clicked it. Now, this still produces a valid .pv file and can be quite useful for testing purposes (e.g. you want to try out some parameters but aren't happy/want to have a look in TRex). However, if your goal is to convert the entire video, you should not do it :-) If you simply do not want to see a window/let it run in the background, just add the `nowindow` flag when starting TGrabs, like so:

```
tgrabs -i video.mp4 -nowindow
```

Which starts without opening a window, so it only runs in the terminal. Hope this answered your question :-)
